lopezMidterm/src/components/Navigation.tsx

```diff
@@ -1,22 +1,28 @@
+import React from 'react';
 import { NavLink } from 'react-router-dom';
 
-function Navigation() {
+const Navigation: React.FC = () => {
   return (
     <nav>
       <ul style={{ listStyle: 'none', display: 'flex', gap: '10px', color: '' }}>
         <li>
-          <NavLink exact to="/" activeclassname="active">
+          <NavLink to="/" activeClassName="active">
```

Comment:
`activeClassName` is from an older version of React Router and doesn't work on version 6. The IDE tells you it's an invalid property as well. Use caution when using other resources and when in doubt look at the official documentation.